compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensions.cpp

```diff
@@ -577,22 +642,25 @@ transform_dialect::VectorWarpDistributionOp::applyToOne(
   // TODO: Hook up into the ApplyPatternOp in CommonExtensions.cpp to
   // automatically get listening capabilities.
 
+  int64_t benefit = 3;
+  MLIRContext *ctx = target->getContext();
+  RewritePatternSet patterns(ctx);
   // MultiReduction lowering is necessary until we have explicit support for
   // distributing that op.
-  if (failed(applyMultiReductionLoweringPatterns(target))) {
-    target->emitOpError("multi-reduction lowering patterns failed to apply");
+  populateMultiReductionLoweringPatterns(target, patterns, benefit--);
```

Comment:
Nit: I'd rather use magic-ish numbers here, `populateFoo(..., /*benefit=*/3)`. The post-decrement trick may backfire if another pack of patterns is added later.